### ISSUE TITLE
Using ssh connect to docker error:Permission denied (publickey,password).

### ISSUE THREAD

**Issue by elttaes** [microsoft/vscode-remote-release#5659]
- VSCode Version: 1.60.2
- Local OS Version: win10
- Remote OS Version: ubuntu20.04(install in docker and only have an port to ssh
- Remote Extension/Connection Type: SSH/Docker
- Logs: OpenSSH_for_Windows_8.1p1, LibreSSL 3.0.2
[10:26:28.113] Resolver error: Error: Permission denied (publickey,password).

Steps to Reproduce:

1.In windows10 I fail to ssh into the ubuntu(install in docker), but I can ssh into the normal ubuntu(not install in docker). And my friend's windows also can't ssh into the ubuntu(docker)
2.But my mac can easyly ssh into the same ubuntu(docker)
3.And I have already update the version of all of the vscode openssh and the ssh plugin but the remote ssh still doesn't work in windows.

Does this issue occur when you try this locally?: Yes
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by tanhakabir** [comment]
Would you be able to send us additional logs from your Windows machine when you try to ssh in? You can look here for how to get the logs: https://code.visualstudio.com/docs/remote/troubleshooting#_reporting-issues

> You can get the Remote - SSH extension logs with Remote-SSH: Show Log from the Command Palette (F1). When reporting Remote - SSH issues, please also verify if you're able to SSH into your machine from an external terminal (not using Remote - SSH).

**Comment by elttaes** [comment]
> If you use PowerShell on the same Windows machine are you able to set up the connection?

Yes,I can connect to the virtual ubuntu using pycharm/powershell/xshell/cmd.exe/vscode's terminal but not Remote-SSH

**Comment by elttaes** [comment]
> > But it seems that Remote-Containers need the physics machine account. I only have the virtual ubuntu's account, so I can't do anything to docker.
> 
> I don't understand what you mean by that, you are using Docker right?
> 
> Otherwise, all I can say is that something is wrong with your configuration and ssh tells us you don't have permission

Hi, thanks for reply! 
This machien is an GPU server, many people need to use it, so everybody have his own virtual ubuntu which installed in the docker. Everyone only have an physics machine port maybe 7000-10000,which mapping to his virtual ubuntu's 22 port. And everyone only have his virtual ubuntu's account/password and the physics machine port in order to using the GPU and the system envirment is separate. 
If my configuration is wrong I think I can't ssh into it using Mac's ssh-Remote, so I think it may occur something that Windows vscode-remote can't connect the port which not is 22 or not is  physics machine.
